TheMovieDB/TheMovieDBTests/NetworkProviderTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  NetworkProviderTests.swift
+//  TheMovieDBTests
+//
+//  Created by Mariana Andrea Orman Berch on 7/10/21.
+//
+
+import XCTest
+@testable import TheMovieDB
+
+class NetworkProviderTests: XCTestCase {
+    
+    var movies: [Movie] = []
+    
+    var imageView: UIImageView = UIImageView()
+    
+    func getMoviesTest() throws {
+        
+        NetworkProvider().getMovies(completion: { [weak self] result in
```

Comment:
you are using two times the network provider init instead create an instance class outside the test methods